docs/development/tutorial/README.md

```diff
@@ -23,8 +24,14 @@
 
 ## Running the code
 
-- IDE (there are launchers checked in for convenience)
-- CLI: `./gradlew :game-app:game-headed:run`
+To build and run the TripleA program (the one you play the game with) either
+
+- use IDE target `game-headed` (there are launchers checked in for convenience)
```

Comment:
Is this more specifically the 'gradle target'? If so, the parenthetical comment should be a new bullet point as that is an alternative way to run the code. Using the checked in launched is the preferred way to launch the app (contains all of the parameters and config needed, if we use it then we'll be likely to update it and check it in for everyone else, if anything changes then checking in a config cascades those updates to all)

Reply:
I intended to give the name of the launcher/target as presented by the IDE (there are AFAIK three), so the developer in spe knows which one to run and doesn't have to guess.  This is not (directly) related to any "gradle target" or a CLI command line.

Reply:
One note of caution with specific launcher names, it can make the documentation more easily out of date. Hence the mention there are launchers is slightly vague, but would be accurate enough if more are added or any renamed.

I've viewed CLI as a bit of a fallback and something for full verification. Hence the getting started document has been geared towards IDE setup.